projects/js-packages/licensing/components/activation-screen-error/use-get-error-content.tsx

```diff
@@ -0,0 +1,185 @@
+import { getRedirectUrl } from '@automattic/jetpack-components';
+import { ExternalLink } from '@wordpress/components';
+import { createInterpolateElement } from '@wordpress/element';
+import { __, sprintf } from '@wordpress/i18n';
+import React from 'react';
+import { LICENSE_ERRORS } from './constants';
+
+type LicenseErrorKeysType = keyof typeof LICENSE_ERRORS;
+type LicenseErrorValuesType = ( typeof LICENSE_ERRORS )[ LicenseErrorKeysType ];
+
+export const UseGetErrorContent = ( licenseError: string, errorType: LicenseErrorValuesType ) => {
+	const hasLicenseError = licenseError !== null && licenseError !== undefined;
+
+	if ( ! hasLicenseError ) {
+		return {
+			errorMessage: null,
+			errorInfo: null,
+		};
+	}
+
+	const needHelpGetInTouchLink = createInterpolateElement(
+		__( 'Need help? <a>Get in touch</a>.', 'jetpack' ),
+		{
+			a: (
+				<ExternalLink
+					href={ getRedirectUrl( 'jetpack-support-license-activation' ) }
+					rel="noopener noreferrer"
+				></ExternalLink>
+			),
+		}
+	);
+
+	switch ( errorType ) {
+		case LICENSE_ERRORS.NOT_SAME_OWNER:
+			return {
+				errorMessage: __(
+					'The account that purchased the plan and the account managing this site are different.',
+					'jetpack'
+				),
+				errorInfo: (
+					<>
+						<p>
+							{ createInterpolateElement(
+								__( 'Follow these <a>steps</a> to resolve it.', 'jetpack' ),
+								{
+									a: (
+										<ExternalLink
+											rel="noopener noreferrer"
+											href={ getRedirectUrl( 'jetpack-support-activate-license' ) }
```

Comment:
Since we are telling them to follow "these steps", i think the general support page is the wrong place to link to here. If we really want to link to a support page, we should probably link directly to the part of that page that deals with this error (i.e. https://jetpack.com/support/activate-a-jetpack-product-via-license-key/#different-user)

Reply:
Yes good point, thanks!